boefjes/boefjes/plugins/kat_retirejs_finding_types/main.py

```diff
@@ -0,0 +1,11 @@
+from typing import List, Tuple, Union
+
+import requests
+
+from boefjes.job_models import BoefjeMeta
+
+
+def run(boefje_meta: BoefjeMeta) -> List[Tuple[set, Union[bytes, str]]]:
+    response = requests.get("https://raw.githubusercontent.com/RetireJS/retire.js/v3/repository/jsrepository.json")
```

Comment:
If we fetch this URL, what do we need the `octopoes/bits/retire_js/retirejs.json` and `rocky/data/retirejs.json` files for?

Reply:
> the one in rocky can be removed after removing finding types from knowledge base

Let's do that in: https://github.com/minvws/nl-kat-coordination/issues/1113